src/Processors/Transforms/WindowTransform.cpp

```diff
@@ -2728,13 +2728,27 @@ void registerWindowFunctions(AggregateFunctionFactory & factory)
                 parameters);
         }, properties}, AggregateFunctionFactory::Case::Insensitive);
 
+    factory.registerFunction("denseRank", {[](const std::string & name,
```

Comment:
BTW, did you looked into `registerAlias` instead of `registerFunction`?

https://github.com/ClickHouse/ClickHouse/blob/37210d29ee0b79b90166721755b21ceea8166ec6/src/Common/IFactoryWithAliases.h#L51

Reply:
Oh, thanks for pointing that out. I wasn't aware of that. I just switched that out for `registerAlias()`. Also made camelCase the default and added snake_case as an alias. Seems to work.

Reply:
BTW, in that case we need to swap `AggregateFunctionFactory::Case::Insensitive` and `AggregateFunctionFactory::Case::Sensitive` because now they are wrong

Reply:
Thanks for the catch. Just fixed it. I removed the argument from registerFunction (now relying on the default Sensitive value) to stay consistent with the rest of the code.